packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -356,12 +447,16 @@ LightCurveViewer.wrappedComponent.propTypes = {
     xOffsetY: PropTypes.number,
     yOffsetX: PropTypes.number,
     yOffsetY: PropTypes.number,
-  })
+  }),
+  
+  // Store-mapped Properties
+  interactionMode: PropTypes.string,
```

Comment:
This could be an enum checking for specific string values instead, like `PropTypes.oneOf(['annotate', 'move'])`

Reply:
👌 Noted, I can make this change, but let me know first if just replacing `interactionMode` with `annotate` improves readability for you.